src/pages/docs/src/codegen/gen-bundlers.js

```diff
@@ -0,0 +1,28 @@
+const { generated, getName, getBundlers } = require('./paths');
+const { render } = require('./markdown');
+const { generateStandardPage } = require('./pages');
+
+function getRoute(name) {
+  return (name && `/tooling/${name}`) || '';
+}
+
+module.exports = function() {
+  const bundlers = getBundlers();
+
+  const imports = bundlers.map(file => {
+
+    const { mdValue, meta = {} } = render(file, generated);
+    const pathElements = file.split('\\').length;
+    const nameToUse = file.split('\\')[pathElements - 2];
```

Comment:
This is not a sound solution. The filename is OS dependent. Either you first normalize to `/` and then split on `/` or you use the OS Separator to perform the splitting.

Also you perform two split operations where only one should be. If you want to avoid a temp. variable just use `filter`:

```ts
const { sep } = require('path');
const [nameToUse] = file.split(sep).filter((_, i, arr) => i === arr.length - 2);
```

Reply:
adapted